model/shared/src/main/scala/explore/model/TargetListGroup.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.Eq
+import cats.implicits._
+import explore.model.decoders._
+import io.circe.Decoder
+import lucuma.core.model.Observation
+import lucuma.core.model.Target
+import lucuma.core.model.TargetEnvironment
+import monocle.Focus
+import monocle.Lens
+
+import scala.collection.immutable.SortedSet
+
+case class TargetListGroup(
+  obsIds:       SortedSet[Observation.Id],
+  targetEnvIds: SortedSet[TargetEnvironment.Id],
```

Comment:
Should this be a `NonEmptySet` ? Or is it used only for unmoored environments?

Reply:
It could be a NonEmptySet, but since it is going to be inside another SortedSet (for the expanded ids) it has to have an ordering. I don't think a NonEmptySet does and I didn't see another NonEmptySortedSet with a brief look. Of course, the next question is: do the expanded ids need to be sorted sets?

Reply:
Perfect. I didn't realize that. I'll make it a NonEmptySet in the next PR.